third-party/rsutils/py/pyrsutils.cpp

```diff
@@ -63,4 +64,23 @@ PYBIND11_MODULE(NAME, m) {
         .def( py::self >= py::self )
         .def( py::self > py::self )
         .def( "is_between", &version::is_between );
+
+    using stabilized_value = rsutils::number::stabilized_value< double >;
+    auto not_empty = []( stabilized_value const & self ) {
+        return ! self.empty();
+    };
+    auto to_string = []( stabilized_value const & self ) -> std::string {
```

Comment:
those 2 functions are global will generic names but have a specific implementation.
What will happen the next time you want to add them to a different utility?
Can you use some namespace or rename to a more specific var names?

Reply:
Global? No, they're just in this code.
If needed, we'll rename or move inline.